SR2024/2023-11-22-december-tech-day-is-in-boldrewood.md

```diff
@@ -0,0 +1,19 @@
+---
+to: Teams who have places at the December Tech Day
+reply-to: "9th December Tech Day Confirmation"
+---
+
+Hi $TODO_CONTACT_GIVEN_NAME,
+
+Just a quick update about the December [Tech Day in Southampton][event-page].
+
+We have now confirmed that this event will be taking place on the University's Boldrewood campus.
```

Comment:
Suggestion: It might be explicitly noting that the location is very close to the previous location, and also that the full address has been updated on the website.

Reply:
There was no previous location.

Reply:
5e67334 points people at the website.